.github/CODEOWNERS

```diff
@@ -396,6 +396,7 @@
 # Story project.
 /core/controllers/story*.py @aks681
 /core/domain/story*.py @aks681
+/core/domain/subtopic_*.py @aks68
```

Comment:
Change it to `/core/domain/subtopic*.py` and move this to the `Topic project` section.

Reply:
Done!